terraform/aws/variables.tf

```diff
@@ -43,9 +43,20 @@ variable "public_subnet_cidr_block" {
   default     = "10.0.0.0/24"
 }
 
-variable "private_subnet_cidr_block" {
-  description = "Cidr block for private Mesos subnet."
-  default = "10.0.1.0/24"
+variable "zones" {
+    default = {
+        zone0 = "eu-west-1a"
```

Comment:
for styling / consistency might be worth either keeping all of these like ```zone0``` or moving to ```zone-0``` like the master_ips var. Otherwise move the master_ips to be like ```master1```